be/src/io/cache/block_file_cache.cpp

```diff
@@ -2077,6 +2077,17 @@ std::string BlockFileCache::clear_file_cache_directly() {
     int64_t normal_queue_size = _normal_queue.get_elements_num(cache_lock);
     int64_t disposible_queue_size = _disposable_queue.get_elements_num(cache_lock);
     int64_t ttl_queue_size = _ttl_queue.get_elements_num(cache_lock);
+
+    if (config::clear_fd_after_clear_file_cache) {
```

Comment:
no need an extra config, just release them directly after files are deleted.

Reply:
in case of performance and robust regression. we can remove this config if it's stable.

Reply:
it seems there is no performance problem, line 2069 `auto s = _storage->clear(clear_msg);` is blocking and time-consuming job before removing FDs.
to avoid too many knobs, keep it simple if the behavior is determined.
we have too many knobs for file cache, too many.